code/game/objects/items/weapons/material/knives.dm

```diff
@@ -25,6 +25,21 @@
 			M = user
 		return eyestab(M,user)
 
+/obj/item/weapon/material/knife/verb/extract_shrapnel(var/mob/living/carbon/human/H)

```

Comment:
This needs to be logged for the mins, so they can bwoink non-antags that do that without a very good reason.

Reply:
I don't think so. It's meant for usage where other options are unavailable and it's life or death. A decent chance at being able to recover + a similar chance of dying faster is preferable to a slow and painful death.